components/_global-navigation.scss

```diff
@@ -75,6 +119,12 @@ $nav-width: 1024px;
 			width: 80px;
 		}
 
+		&.wds-has-multiple-subitems {
```

Comment:
Will it be/is it done in the base branch (XW-1618)? If these will be dropped I might overwrite `display: flex;` for them back to `block` or `inline-block`.